vNext/shared/AppInsightsCore/src/JavaScriptSDK/AppInsightsCore.ts

```diff
@@ -53,7 +53,7 @@ export class AppInsightsCore implements IAppInsightsCore {
         this.logger = new DiagnosticLogger(config);
 
         // Initial validation
-        extensions.forEach((extension: ITelemetryPlugin) => {
+        this._extensions.forEach((extension: ITelemetryPlugin) => {
```

Comment:
I believe `this._extensions` is an empty array at this point

Reply:
I see, so is actually checking the input, my mistake reverting this change

Reply:
Merged both extensions and config.extension so validation applies to everything